src/recommendations/install-configure.md

```diff
@@ -59,6 +59,14 @@ Save the the `composer.json` file and run:
 composer update magento/product-recommendations --with-dependencies
 ```
 
+### Install Page Builder support
+
+The Product Recommendations content type in Page Builder is supported in Magento 2.3.1 and later. Because Page Builder [does not support](https://devdocs.magento.com/release/#compatibility) the same number of versions as the Product Recommendations module, you need to explicitly install Page Builder support with the following:
```

Comment:
Would it be worth it to mention that it is suggested in the original `composer require magento/product-recommendations` install process? It should look like:

```
composer require magento/product-recommendations
...
magento/product-recommendations suggests installing magento/module-page-builder-product-recommendations (Enable Product Recommendations for PageBuilder. Supported in Magento Commerce 2.3.1 and later.)
```

Reply:
yeah we can mention that as well

Reply:
@rossbrandon I like the idea of integrating this content into the general install section above. I think that flows more.

Reply:
agree!

Reply:
instead of explaining the reason for compatibility, should we just say "Product Recommendations for Page Builder is an optional module and is installed separately"

Reply:
@misha-kotov agreed. I revised that part. Take a look and let me know your thoughts.